src/pages/Home/Home.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setAlert } from '../../slices/appSlice/appSlice';
+import { logout } from '../../slices/authSlice/authSlice';
+import { planesThunk } from '../../slices/planeSlice/planeAPI';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const token = useSelector((state) => state.auth.token);
+  const planes = useSelector((state) => state.planes.planes);
+  useEffect(() => {
+    dispatch(planesThunk(token)).then((res) => {
+      if (res.error) {
+        dispatch(setAlert(res.payload));
+      }
+    });
+  }, [dispatch, token]);
+  return (
+    <div>
+      <h1>Home Page</h1>
+      <button type="button" onClick={() => dispatch(logout())}>Logout</button>
```

Comment:
- [ ] Great job implementing the user logout func, but right new when the user click on the button:
   - The token is removed from localStorage
 But on the window nothing happen:
  - It would be great if reload the page with `window.reload` indicating for the user that he logged successfully, or alert a message for that 😉

Reply:
Actually This is supposed to happen in nav bar so I didn't implement anything and leave it as an example.